app/models/metadata.rb

```diff
@@ -26,6 +26,64 @@ class Metadata < ApplicationRecord
     }
   ].to_json
 
+  def self.filters_to_json
+    metadata = Metadata.all
+    unique_categories = metadata.pluck(:category).compact.uniq.sort
```

Comment:
It looks like this is the only variable used across this method. Do we need the other `unique_` variables?